test/regressions/webpack.config.js

```diff
@@ -32,12 +32,11 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.(js|ts|tsx)$/,
-        // prism.js blocks @mui/internal-markdown/prism from being interpreted as ESM in this build.
-        exclude: /node_modules|prism\.js/,
+        test: /\.(js|mjs|ts|tsx)$/,
+        exclude: /node_modules/,
         loader: 'babel-loader',
         options: {
-          cacheDirectory: true,
+          // cacheDirectory: true,
```

Comment:
I don't understand this line.

The only thing I found is that this property should be a string 🤔https://webpack.js.org/configuration/cache/#cachecachedirectory

Reply:
accidentally committed. I don't think it's doing much for us either way.